pebblo_safeloader/langchain/textloader_postgress/.env.sample

```diff
@@ -1,11 +1,11 @@
 # OpenAI credentials
 OPENAI_API_KEY=<YOUR OPENAI API KEY>
 
-# Pebblo configuration
-PEBBLO_CLOUD_URL=<PEBBLO CLOUD URL>
-PEBBLO_API_KEY=<YOUR PEBBLO API KEY>
-PEBBLO_CLASSIFIER_URL="http://localhost:8000/"
-
 # Postgres configuration
 PG_CONNECTION_STRING = "postgresql://<USERNAME>:<PASSWORD>@<HOST>:<PORT>/<DATABASE-NAME>"
 
+# Pebblo configuration
+PEBBLO_CLASSIFIER_URL="<PEBBLO-SERVER-HOST:PEBBLO-SERVER-PORT>"
```

Comment:
nit: we can add this defaults to `http://localhost:8000/` if unspecified

Reply:
Done